lib/associations/belongs-to-many.js

```diff
@@ -252,7 +252,12 @@ class BelongsToMany extends Association {
     if (this.primaryKeyDeleted === true) {
       targetAttribute.primaryKey = sourceAttribute.primaryKey = true;
     } else if (this.through.unique !== false) {
-      const uniqueKey = [this.through.model.tableName, this.foreignKey, this.otherKey, 'unique'].join('_');
+      let uniqueKey;
```

Comment:
Please add some info about this option in

- https://github.com/sequelize/sequelize/blob/master/docs/associations.md#belongs-to-many-associations
- https://github.com/sequelize/sequelize/blob/master/lib/model.js#L4258